portfolio/src/main/webapp/script.js

```diff
@@ -76,13 +76,12 @@ function getAndDisplayComments() {
       commentContainer.innerHTML += comment.CONTENT + '<br>';
       commentContainer.innerHTML += comment.numLikes + ' likes<br>';
       commentContainer.innerHTML += 'Posted at ' + comment.TIMESTAMP + '<br>';
-      commentContainer.innerHTML += '<button class="btn btn-danger" onclick="addLike(\'' + comment.ID + '\');">Like</button>';
+      const likeButtonHTML = '<form action="/data" method="POST">' + 
+        '<input type="hidden" name="comment-id" value="' + comment.ID + '" />' + 
+        '<input type="submit" value="Like" class="btn btn-danger btn-sm" /></form>';
```

Comment:
Hard coding HTML in my js file doesn't seem like the cleanest way to do this but I couldn't get it to work exactly how I wanted when using document.createElement functions. Considering adding a TODO to come back to this later this week?